reference.rb

```diff
@@ -0,0 +1,66 @@
+class MoviesController < ApplicationController
+  def index
+    @movies = Movie.order(upvotes: :desc)
+  end
+
+  def show
+    id = params[:id]
+    @movie = Movie.find(id)
+  end
+
+  def new
+    @movie = Movie.new
+  end
+
+  def create
+    @movie = Movie.create(movie_params[:movie])
+    if @movie.save
+      redirect_to movie_path(@movie)
+    else
+      render :new
+    end
+  end
+
+  def edit
+    id = params[:id]
+    @movie = Movie.find(id)
+  end
+
+  def update
+    id = params[:id]
+    @movie = Movie.find(id)
+    @movie.update(
+    name: movie_params[:movie][:name],
```

Comment:
For readability, I would indent lines 33-35 so that you can more easily see that it is nested within line 32